sweep/txgenerator.go

```diff
@@ -315,7 +320,7 @@ func createSweepTx(inputs []input.Input, outputs []*wire.TxOut,
 		estimator.parentsWeight,
 	)
 
-	return sweepTx, nil
+	return sweepTx, txFee, nil
```

Comment:
Do we also want to return information w.r.t: the fee desired by the input, vs the total fee that was satisficed by the sweeper?

Reply:
Don't think we can do it here - it just creates a tx using the specified feerate, so by the time we arrived here, the feerate has already been determined by previous steps.